Case/src/Program.cs

```diff
@@ -11,58 +12,60 @@ class Program
 {
     static void Main()
     {
-        FileReader fileReader = new FileReader("export");
-        List<string>? files = fileReader.ReadDirectory();
-        if (files == null)
-        {
-            Console.WriteLine("There's no files at the selected directory");
-            return;
-        }
+        FileReader fileReader = new FileReader("pending");
+        List<string>? files = fileReader.ReadList();
+        
         ItemScanner itemScanner = new ItemScanner();
+        StorageSystem storage = new StorageSystem();
 
         FileMover fileMover = new FileMover(files);
 
         ColorHandle colorHandle = new ColorHandle();
-
-        PluckListPrinter pluckListPrinter;
-        OptionPrinter optionPrinter = new OptionPrinter();
         
         char readKey = ' ';
         int index = -1;
         PluckList? pluckList = null;
         List<Item> scannedItems;
-        
+
+        FileReader xmlsFileReader = new FileReader("allPluckLists");
+        List<string> xmlFiles = xmlsFileReader.ReadList();
+        CSVWriter csv = new CSVWriter("items.csv");
+
+        List<Item> sortedItems = new List<Item>();
+        foreach (string xml in xmlFiles)
+        {
+            List<Item> items = new XMLReader(xml).Read<PluckList>().Lines;
```

Comment:
Can use PluckList.Deserialize(). Also, `Read` may return null.
```suggestion
            List<Item> items = PluckList.Deserialize(xml)?.Lines;
```